### PR TITLE
Add example data to new courses

### LINKED ISSUE #99: Default Units & Lessons
When we creating a new course, we need to populate that course with [some default units and lessons](https://docs.google.com/document/d/16luvkp6r4Rlh-WSZlLvIrtm8ue1XT_k7/edit). This helps provide some structure to a new course and guide the course contributor when collecting vocab items.

[Link to a doc containing the default unit and lessons](https://docs.google.com/document/d/16luvkp6r4Rlh-WSZlLvIrtm8ue1XT_k7/edit)

Steps:
1. Add the default unit and lessons to a JSON file in the `api` folder.
2. Update the POST '/course' endpoint to generate the default units and lessons using the data from the API folder.

Don't forget:
1. Update existing Jest tests for the POST '/course' endpoint, and create an assertion to test that the default units and lessons are properly created.

### PR DESCRIPTION
**Pull request by rustom**
## Description

Adds hardcoded example units, lessons, and vocab items to any new course that is created. Fixes #99.

### CODE REVIEW

File: api/src/api/language/course.js [hunk 1]
```diff
@@ -32,6 +39,69 @@ router.patch(
   }),
 );
 
+/**
+ * Uploads example units, lessons, and vocab items for any new course that is created.
+ * @param {course_id} course_id of the new course that was just created
+ */
+async function populateExampleData(course_id) {
```
Comment (ashayp22) [inline api/src/api/language/course.js:46]: to utils

File: api/src/api/language/course.js [hunk 2]
```diff
@@ -32,6 +39,69 @@ router.patch(
...
+  for (const unit of exampleData.units) {
+    // Get data for the unit from exampleData
+    const unitData = unit['unitData'];
+
+    // Update example unit data with IDs and order
+    const order = await getNumUnitsInCourse(course_id);
```
Comment (ashayp22) [inline api/src/api/language/course.js:52]: break up modularize

File: api/src/api/language/course.js [hunk 3]
```diff
@@ -32,6 +39,69 @@ router.patch(
...
+    unitData['_course_id'] = course_id;
+    unitData['_order'] = order;
+
+    // Create and save new unit
+    const newUnit = new models.Unit(unitData);
+    await newUnit.save();
```
Comment (ashayp22) [inline api/src/api/language/course.js:58]: check if unit id is valid

Comment (ashayp22) [reply api/src/api/language/course.js:58]: course id, my bad rustom

**Review by ashayp22 — CHANGES_REQUESTED** [review]
lfg

File: api/src/utils/languageHelper.js [hunk 4]
```diff
@@ -110,12 +115,12 @@ const updateDocumentInTransaction = async (model, document, session) => {
   The methods below determine if a list of ids for a course, unit, and/or lesson is valid, meaning that all of the ids
   are valid ObjectIds and a document exists in the appropriate collection for each _id. 
 */
-module.exports.checkIds = async ({
+async function checkIds({
```
Comment (ashayp22) [inline api/src/utils/languageHelper.js:118]: wondering why you made this change

Comment (rustom) [reply api/src/utils/languageHelper.js]: reverted

**Review by ashayp22 — APPROVED** [review]

**Pull request merged**